packages/web/src/components/gcds-checkbox/gcds-checkbox.css

```diff
@@ -21,41 +21,48 @@
 
 @layer default {
   :host .gcds-checkbox {
-    max-width: 90%;
+    position: relative;
+    max-width: var(--gcds-checkbox-max-width);
+    min-height: calc(
+      var(--gcds-checkbox-input-height-and-width) - var(--gcds-checkbox-padding)
+    );
     font: var(--gcds-checkbox-font);
     color: var(--gcds-checkbox-default-text);
     margin: var(--gcds-checkbox-margin) !important;
+    padding: var(--gcds-checkbox-padding) 0 0;
```

Comment:
Should we include the additional `0 0` in the `--gcds-checkbox-padding` token?

Reply:
I didn't include the `0 0` to use the padding token to calculate the correct min-height for the checkbox.